chia/server/upnp.py

```diff
@@ -65,7 +65,7 @@ def shutdown(self):
             return
         self.queue.put(("shutdown",))
         log.info("UPnP, shutting down thread")
-        self.thread.join()
+        self.thread.join(5)
```

Comment:
this seems like an unrelated change. Is there a good reason to do this? I would expect a comment in the code to explain why we need to detach the thread after 5 seconds

Reply:
we should probably fix the issue causing the UPnP thread to hang instead. This will just hide it